### PR TITLE
More Flash Messages

### PR DESCRIPTION
**Pull request by JVickery-TBS**
Dupe of https://github.com/ckan/ckan/pull/7996 but just for our 2.9 fork/branch

### CODE REVIEW

File: ckan/views/group.py [hunk 1]
```diff
@@ -958,7 +964,13 @@ def post(self, group_type, is_organization, id=None):
...
             if id != group['name']:
                 _force_reindex(group)
-
+            if group_type == u'organization':
+                h.flash_success(_(u'Organization updated.'))
+            elif group_type == u'group':
+                h.flash_success(_(u'Group updated.'))
+            else:
+                h.flash_success(
+                    _(u'%s updated.') % _(group_type.capitalize()))
```
Comment (wardi) [inline ckan/views/group.py:973]: We're not using other group types and this isn't translatable the way it's written. Remove the else?

File: ckan/views/group.py [hunk 2]
```diff
@@ -1109,6 +1121,8 @@ def post(self, group_type, is_organization, id=None):
 
         try:
             group_dict = _action(u'group_member_create')(context, data_dict)
+            h.flash_success(_(u'Assigned %s as %s.') % (
+                data_dict['username'], data_dict['role']))
```
Comment (wardi) [inline ckan/views/group.py:1125]: role can't be translated this way, use different `_(strings)` based on the role instead?

File: ckan/views/group.py (hunk 1 above)
Comment (JVickery-TBS) [reply ckan/views/group.py]: Okay, just removed the group type condition and doing `is_organization` for these messages.

File: ckan/views/group.py (hunk 2 above)
Comment (JVickery-TBS) [reply ckan/views/group.py]: Using `authz.trans_role` now. does that work??

Comment (wardi) [reply ckan/views/group.py]: Wow, I hate the way trans_role is written, and I bet it's going to result in incorrect capitalization, but I guess it works

Comment (JVickery-TBS) [reply ckan/views/group.py]: Hahaha, just making these changes to the upstream PR too. Looks like `trans_role` is written differently there: https://github.com/ckan/ckan/blob/master/ckan/authz.py#L269

Comment (wardi) [reply ckan/views/group.py]: Yeah looks like Sergey cleaned it up. But still, `trans_role` returns the wrong capitalization (and maybe other problems like masculine/feminine) to insert into a sentence

Comment (JVickery-TBS) [reply ckan/views/group.py]: Yeah, would be nice to have a full humanization implementation for roles

Comment (wardi) [reply ckan/views/group.py]: Could use a dict lookup here to solve the issue. i.e
`{'member': _(u'Assigned %s as a member.'), ...}`

Comment (JVickery-TBS) [reply ckan/views/group.py]: Yeah, reckon that is the best thing to do for now. I have done that here and upstream then

**Comment by JVickery-TBS** [comment]
@wardi are you able to review this again? Thanks!

**Review by wardi — APPROVED** [review]

**Pull request merged**